utils/install-redirect.js

```diff
@@ -2,6 +2,8 @@ import { REPO_TYPE, REPO, CHART, VERSION } from '@/config/query-params';
 
 export default function(product, chartName, defaultResourceOrRoute) {
   return async function middleware({ redirect, store } ) {
+    const cluster = store.getters['currentCluster'].id || 'local';
```

Comment:
I hit an error, not sure if it's something that can/should be solved here though? 

Logging in as a user who's a `Cluster Member` of single non-local cluster results in an empty home page cluster list and no clusters listed in the side nav (same clusters that are missing here show up if the user is admin). Clicking on 'Continuous Delivery` then results in 
```
Cannot read property 'id' of undefined
    at _callee$ (install-redirect.js?8cf0:5)
```
Guessing currentCluster is never set as no clusters are visible?  For this use case falling back to the local user wouldn't help either. 

I'm not sure why the cluster is not appearing in `v1/management.cattle.io.cluster`, but is this something we need to guard against?

Reply:
@richard-cox I've updated the code for this